.github/workflows/release.yaml

```diff
@@ -91,14 +92,6 @@ jobs:
             mv inputs.json.tmp inputs.json
           done
           echo "release-image-inputs=$(cat inputs.json | tr -d '\n')" >> $GITHUB_OUTPUT
-      # Sleep a random amount of time up to 10 minutes.
-      # This staggers our pushes, to hopefully avoid hitting rate limits.
-      # See https://github.com/chainguard-images/images/issues/173
-      - name: Random Sleep
-        run: |
-          rand=$(( ( RANDOM % 600 )  + 1 ))
-          echo "sleeping $rand seconds..."
-          sleep $rand
```

Comment:
We'll still need this if we're pushing to GHCR, or else the workflows will fail :(

Reply:
ill put the sleep right before this tagging process